.changeset/thin-bears-smile.md

```diff
@@ -0,0 +1,8 @@
+---
+"wrangler": minor
```

Comment:
Is this a minor? It's a feature for a beta feature of Wrangler so I think it should be a patch?